doc/api/repl.markdown

```diff
@@ -106,6 +108,14 @@ the following values:
 You can use your own `eval` function if it has following signature:
 
     function eval(cmd, context, filename, callback) {
+      var result;
+      try {
+        result = vm.runInThisContext(cmd);
+      } catch (e) {
+        if (isRecoverableError(e)) {
```

Comment:
@blakeembrey To think of it, this part

```js
    if (e.message === 'Missing } in template expression') {
      return true;
    }
```

may not be readily understandable for the users. So, lets remove that part alone.